### ISSUE TITLE
Sub instantly sinks on starting in Gazebo

### ISSUE THREAD

**Issue by ananmayjain** [patrickelectric/bluerov_ros_playground#34]
Hi,

I'm running ubuntu18 for this. When I launch the sub through roslaunch, the sub instantly sinks, and keeps sinking. No commands I publish to mavros seem to work. What should I do?

**Comment by PabloHebe** [comment]
I have the same problem. Seems to be that something is wrong with freefloating_gazebo? When I added a box it also instantly dropped.

**Comment by kurransingh** [comment]
I am also experiencing this issue. Did either of you end up figuring this out?

**Comment by patrickelectric** [comment]
Is everybody here using the versions that are in the README ?

**Comment by kurransingh** [comment]
Yes, I am using ROS Kinetic and I followed the links from the README to freefloating-gazebo and from there to freefloating-gazebo-demo. I followed the build instructions from the README with no issues. 
If I set the buoyancy of the BlueROV to 0 for all four sections of the vehicle and turn gravity off the vehicle stays in place properly... 
Thanks in advance for any help you might be able to give!
